batch/batch/server/server.py

```diff
@@ -114,10 +117,47 @@ def _next_task(self):
     def _has_next_task(self):
         return self._task_idx < len(self._tasks)
 
+    def _create_pvc(self):
+        pvc = v1.create_namespaced_persistent_volume_claim(
```

Comment:
Will this need any additional permissions?

Reply:
ah yeah, probably, but oddly the tests do not fail.